telemetry/definitions/commonDefinitions.json

```diff
@@ -199,63 +205,121 @@
         {
             "name": "apigateway_copyUrl",
             "description": "Copying an API Gateway remote URL",
-            "metadata": [{ "type": "result" }]
+            "metadata": [
```

Comment:
not a fan of the amount of newlines the formatter generated here. however, I don't think the existing format is standard in any way. @hunterwerlla easy way to lint for the existing style? otherwise probably stick with the style in this change.

Reply:
I used the prettier code formatter for this change. Is there a certain preferred formatter?

Reply:
we should move our prettier settings to the root, the reason it's formatting incorrectly is probably because prettier was run manually

Reply:
@rli / @hunterwerlla  Which formatting is the desired approach, and what form would you like this PR to take?

Reply:
I am fine as-is, but we should play with the prettier settings soon to improve the output

Reply:
Reverting back to the original format for now.